### ISSUE TITLE
pmod_led does not match the Tang Primer 25K pinout

### ISSUE THREAD

**Issue by harbaum** [sipeed/TangPrimer-25K-example#11]
The pin assignments of the pmod_led demo does not match the Tang Primer 25K. E.g. the "key" in this example is assigned to K6. But the keys on Tang Primer 25K base board are on H10 and H11. K6 is USB_N 

https://github.com/sipeed/TangPrimer-25K-example/blob/434ef838e20836d6cc7e07b1dd1994d0de4a74e2/pmod_led/src/pmod_led.cst#L69

Furthermore the button S1 on H11 is active high and does not have a pull-down. The pull-down in the FPGA thus needs to be enabled and the polarity reversed.

**Comment by zavs** [comment]
@harbaum According to the schematic, s1 connects with H11 and s2 with H10. Also, it seems that the LXES15AAA1 will give them a GND bias, I guess. 
<img width="227" alt="image" src="https://github.com/sipeed/TangPrimer-25K-example/assets/3454120/729d1e78-37ad-444a-8835-3f439395fef2">

**Comment by harbaum** [comment]
Yes, H10 and H11, that's what I wrote. And no, D1 and D2 don't give a noticable GND bias. They bascially latch to the high state once pressed without pulldown enabled in the FPGA.

I am sure C1 and C2 will discharge over time. But that does not happen within a few seconds. Didn't try to wait longer ...

**Comment by harbaum** [comment]
The problem is that these demos probably are what users try first. E.g. the HDMI demo also does not work for me. And it also has a misconfigured LED. So without getting an HDMI output I checked if I at least the LED would be blinking as that's what I expected from a first glance at the code. It didn't! So I assumed something fundamenally wrong with my board. An the fact that this pmod_led  demo also didn't do anything even supported that assumption.

**Comment by martinloretzzz** [comment]
PR: https://github.com/sipeed/TangPrimer-25K-example/pull/16
